test/resources/case33_empty_label_handling/empty-labels-config.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: policy.open-cluster-management.io/v1
+kind: ConfigurationPolicy
+metadata:
+  name: case33-empty-labels
+spec:
+  remediationAction: enforce
+  severity: high
+  object-templates:
+  - complianceType: musthave
+    objectDefinition:
+      apiVersion: v1
+      kind: Node
+      metadata:
+        name: test-managed-control-plane
+        labels:
+          cluster.ocs.openshift.io/openshift-storage: ""
```

Comment:
As a sanity check, could we also check using a policy that sets `cluster.ocs.openshift.io/openshift-storage: null` to make sure the policy doesn't remove the field? (Though I'm also uncertain what the behavior would be there in terms of compliance...)

Reply:
It would be nice to have a test just to check what this currently does, and understand if that behavior has changed (or if we change it accidentally in the future)

Reply:
Currently, having an unset label/setting it to `nil` results in the deletion of **all** the labels attached to the object. I'll probably make a separate ticket to address this issue as it's a bit more involved and possibly affect annotations as well (need to confirm this but they both use the same helper function to populate the fields).